stubs/SQLAlchemy/@tests/stubtest_allowlist.txt

```diff
@@ -126,1067 +42,23 @@ sqlalchemy.sql.annotation.Annotated.__new__
 # as having conflicting __new__/__init__ methods
 sqlalchemy.orm.unitofwork.PostSortRec.__new__
 
-# unclear problems
+# KeyError/AttributeError on import due to dynamic initialization from a different module
+sqlalchemy.testing.fixtures
+sqlalchemy.testing.pickleable
+sqlalchemy.testing.plugin.bootstrap
+
+# method arguments starting with double underscores in the implementation trips up stubtest
```

Comment:
Interesting, this could be a stubtest bug. Maybe file a mypy issue if it is?

Reply:
Probably causes confusion because in typing this means pos-only args?

Reply:
My suspicion is that stubtest's logic to do with name-mangling might be going wrong somehow. But I don't know, haven't looked into this at all.

Reply:
https://github.com/python/mypy/issues/14560